go/tasks/pluginmachinery/io/iface.go

```diff
@@ -51,6 +61,8 @@ type OutputFilePaths interface {
 	// A Fully qualified path (URN) where the error information should be placed as a protobuf core.ErrorDocument. It is not directly
 	// used by the framework, but could be used in the future
 	GetErrorPath() storage.DataReference
+
+	OutputDataSandbox
```

Comment:
nit: Can you move it to the top of the interface... I find it a bit more readable..

Reply:
Sure